apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -1501,8 +1507,80 @@ export class RushConfiguration {
    * @param variant - The name of the current variant in use by the active command.
    */
   public getCommonVersions(variant?: string | undefined): CommonVersionsConfiguration {
-    const commonVersionsFilename: string = this.getCommonVersionsFilePath(variant);
-    return CommonVersionsConfiguration.loadFromFile(commonVersionsFilename);
+    if (!this._commonVersionsConfigurations) {
+      this._commonVersionsConfigurations = new Map();
+    }
+
+    const variantKey: string = variant || '';
```

Comment:
Isn't the default variant called `default` (from the comments on `_commonVersionsConfigurations` and `_implicitlyPreferredVersions`)?

Reply:
Good catch, this is an out-of-date comment. Will update